components/Layout/NavListOfLinks.tsx

```diff
@@ -0,0 +1,61 @@
+import { FC } from "react";
+import {
+  Flex,
+  Link,
+  useColorMode,
+  NAMED_COLORS,
+  Center,
+  useBreakpointValue,
+} from "@ironfish/ui-kit";
+import { OuterReferenceIcon } from "svgx";
+
+const NavListOfLinks: FC = () => {
+  const { colorMode } = useColorMode();
+  const isDarkMode = colorMode === "dark";
+  const style = useBreakpointValue({
+    base: {
+      link: {
+        fontSize: "2.3125rem",
+      },
+      container: {
+        display: { base: "flex", lg: "none" },
+        flexDirection: "column",
+        w: "100%",
+        h: "100vh",
+        p: "2.5rem 0rem 2.5rem 2rem",
+        bgColor: isDarkMode ? NAMED_COLORS.LIGHT_BLACK : NAMED_COLORS.WHITE,
+      },
+    },
+    lg: {
+      link: {
+        fontSize: "0.875rem",
+        mr: "2rem",
+        whiteSpace: "nowrap",
+      },
+      container: {
+        display: { base: "none", lg: "flex" },
+      },
```

Comment:
First of all, add this component props.
`const NavListOfLinks: FC<FlexProps> = (props) => {`
Then replace `<Flex sx={style.container}>` with `<Flex {...props}>`
After that move customization of container to parents